backend/collaboration-service/app/src/routes/room.ts

```diff
@@ -13,10 +14,22 @@ roomRouter.use((_req: Request, _res: Response, next: NextFunction) => {
   next();
 });
 
-roomRouter.post("/", async (_: Request, res: Response) => {
+roomRouter.post("/", async (req: Request<{}, {}, { difficulty: string; topic: string; }>, res: Response) => {
+
   const roomId = v4();
 
-  const { code, data, status } = createRoom(roomId);
+  const { difficulty, topic } = req.body;
```

Comment:
same thing, difficulty and topic is an array

Reply:
Resolved in [8304b6a](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/24/commits/8304b6ab4ab3e94be379ff3bea271b3f0f8d79b5)